lib/ansible/modules/cloud/vmware/vmware_dvswitch_netflow.py

```diff
@@ -0,0 +1,273 @@
+            results['collector_port_previous'] = netflow_config.collectorPort
+        if netflow_config.observationDomainId != self.netflow_observation_domain:
+            changed = True
+            changed_list.append("observation domain ID")
+            results['observation_domain_previous'] = netflow_config.observationDomainId
+        # looks like switch IP can't be set via SOAP API!?
+        if netflow_config.activeFlowTimeout != self.netflow_active_flow_export:
+            changed = True
+            changed_list.append("active flow export timeout")
+            results['active_flow_export_timeout_previous'] = netflow_config.activeFlowTimeout
+        if netflow_config.idleFlowTimeout != self.netflow_idle_flow_export:
+            changed = True
+            changed_list.append("idle flow export timeout")
+            results['idle_flow_export_timeout_previous'] = netflow_config.idleFlowTimeout
+        if netflow_config.samplingRate != self.netflow_sampling_rate:
+            changed = True
+            changed_list.append("sampling rate")
+            results['sampling_rate_previous'] = netflow_config.samplingRate
+        if netflow_config.internalFlowsOnly != self.netflow_process_internal_flows:
+            changed = True
+            changed_list.append("internal flows")
+            results['process_internal_flows_only_previous'] = netflow_config.internalFlowsOnly
+        config_spec.ipfixConfig = netflow_config_spec
+
+        if changed:
+            if self.module.check_mode:
+                changed_suffix = ' would be changed'
+            else:
+                changed_suffix = ' changed'
+            if len(changed_list) > 2:
+                message = ', '.join(changed_list[:-1]) + ', and ' + str(changed_list[-1])
+            elif len(changed_list) == 2:
+                message = ' and '.join(changed_list)
+            elif len(changed_list) == 1:
+                message = changed_list[0]
+            message += changed_suffix
+            if not self.module.check_mode:
+                try:
+                    task = self.dvs.ReconfigureDvs_Task(config_spec)
+                    wait_for_task(task)
+                except TaskError as invalid_argument:
+                    self.module.fail_json(
+                        msg="Failed to update DVS : %s" % to_native(invalid_argument)
+                    )
+        else:
+            message = "NetFlow already configured properly"
+        results['changed'] = changed
+        results['result'] = message
+
+        self.module.exit_json(**results)
+
+
+def main():
+    """Main"""
+    argument_spec = vmware_argument_spec()
+    argument_spec.update(
+        dict(
+            switch=dict(required=True, aliases=['dvswitch']),
+            collector_ip=dict(type='str', default=''),
+            collector_port=dict(type='int', default=0),
+            observation_domain=dict(type='int', default=0),
+            # switch_ip=dict(type='str', default=''),
```

Comment:
Could you please remove this commented part ?

Reply:
This is the only option from the GUI that's missing